book/website/communication/citable/citable-cff.md

```diff
@@ -0,0 +1,153 @@
+
+```yaml
+abstract: "This is my awesome research software. It does many things."
+authors:
+  - family-names: John
+    given-names: Doe
+    orcid: "https://orcid.org/0000-0001-8888-9999"
+cff-version: 1.2.0
+date-released: "2021-10-13"
+identifiers:
+  - description: "This is the collection of archived snapshots of all versions of My Research Software"
+    type: doi
+    value: 10.5281/zenodo.123456
+  - description: "This is the archived snapshot of version 0.11.2 of My Research Software"
+    type: doi
+    value: 10.5281/zenodo.123457
+keywords:
+  - "amazing software"
+  - research
+license: Apache-2.0
+message: "If you use this software, please cite it using the metadata from this file."
+repository-code: "https://github.com/citation-file-format/my-research-software"
+title: "My Research Software"
+version: 0.11.2
+```
+
+The complete list of fields is described in the [CFF schema guide](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md). In the next section, you can find out which tools can help you create and use the `CITATION.cff` file.
+
+### Steps to Make Your Software Citable
+
+To make your software citable, you need to follow the three steps below.
+
+#### Step 1. Create a `CITATION.cff` File
+
+There are two ways of creating a `CITATION.cff` file.
+
+1. Edit the file manually in your favorite code editor. The disadvantages of this method are installing the required tools on your system and doing the validation yourself. The error messages of the validation can be relatively long and difficult to understand.
+
+2. Use [cffinit](https://citation-file-format.github.io/cff-initializer-javascript/) which guides you through the process of creating your citation file. The `cffinit` has a few advantages compared to manual editing such as
+
+    - no need for installing extra tools;
+    - no need for automatic validation;
+    - guidance for each fields;
+    - visual feedback to indicate issues.
+
+We suggest using `cffinit` as it simplifies the creation and validation. For more details on using `cffinit` see {ref}`cm-citable-cffinit`.
+
+#### Step 2. Validate Your `CITATION.cff` File
+
+```{note}
+If you used `cffinit` to create your `CITATION.cff`, then you can skip to [step 3](citable-cff.html#add-your-citation-cff-to-a-public-code-repository).
+```
+
+Once you have a `CITATION.cff` file, it needs to be validated to make sure there are no issues. You can validate your `CITATION.cff` file on the command line with the [`cffconvert` Python package](https://pypi.org/project/cffconvert/).
+
+```shell
+cd <directory-containing-your-CITATION.cff>
+cffconvert --validate
+```
+
+If you prefer to use Docker, you can use the [`cffconvert` Docker image](https://hub.docker.com/r/citationcff/cffconvert):
+
+```bash
```

Comment:
The above language is `shell` but it's `bash` here. Not that it really matters, but just flagging it for you. :)

Reply:
made both shell ;)